test/3x/torch/quantization/test_pt2e_quant.py

```diff
@@ -283,3 +282,62 @@ def test_mixed_fp16_and_int8(self, force_not_import_ipex):
         opt_model = torch.compile(converted_model)
         out = opt_model(*example_inputs)
         assert out is not None
+
+    @pytest.mark.skipif(not GT_OR_EQUAL_TORCH_VERSION_2_5, reason="Requires torch>=2.5")
+    @pytest.mark.parametrize("half_precision_dtype", ["fp16", "bf16"])
+    @pytest.mark.parametrize("op_name_or_type", ["conv1", "fc1", torch.nn.Linear, torch.nn.Conv2d])
+    @pytest.mark.parametrize("bias", [True, False])
+    def test_auto_tune_mixed_int8_and_16bits(self, half_precision_dtype, op_name_or_type, bias, force_not_import_ipex):
+        # Just make sure the pattern matches, not the accuracy.
+        # config1: int8 for all
+        # config2: half precision for linear/conv
+        from neural_compressor.torch.quantization.config import INT8StaticQuantConfig
+        from neural_compressor.torch.quantization.autotune import autotune, TuningConfig
+
+        config1 = INT8StaticQuantConfig()
+        config2 = INT8StaticQuantConfig().set_local(
+            op_name_or_type, StaticQuantConfig(w_dtype=half_precision_dtype, act_dtype=half_precision_dtype)
+        )
+        tune_config = TuningConfig(config_set=[config1, config2], tolerable_loss=-0.1)

```

Comment:
what is tuning space here, for each conv/linear, int8/bf16/fp16 are choices? total 4x3 = 12 tuning instance?  is it possible to specify only tuning with int8/bf16 or int8/fp16.  int8/bf16/fp16 is a rare case

Reply:
This UT convers 16 cases. But for each case, the tuning space includes only 2 tuning instances. 
It does not mix the int8/bf16/fp16 in one model, it only mixed int8/bf16 or int8/fp16.